src/ConstraintValidator/Operator/String/StringEndsWithOperatorValidator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Unleash\Client\ConstraintValidator\Operator\String;
+
+/**
+ * @internal
+ */
+final class StringEndsWithOperatorValidator extends AbstractStringOperatorValidator
+{
+    protected function validate(string $currentValue, array|string $searchInValue): bool
+    {
+        assert(is_string($searchInValue));
+
+        return str_ends_with($searchInValue, $currentValue);
```

Comment:
Is `str_ends_with` supported in PHP 7.2+, or is it only PHP 8.0?

Reply:
https://www.codegrepper.com/code-examples/php/str_ends_with+php+7

Reply:
8.0, but this repo contains a downgrader which downgrades the code to multiple version, it gets automatically changed to something suitable for each version during deploy, see for example [here](https://github.com/Unleash/unleash-client-php/commit/f0a215569dba6260c44f56f88c30f42cb8cd0433) for version downgraded to php 7.4.

Reply:
And here you can see the str_* functions downgraded: https://github.com/Unleash/unleash-client-php/blob/f0a215569dba6260c44f56f88c30f42cb8cd0433/src/Configuration/UnleashConfiguration.php#L59

Reply:
This is nice. Thanks for explaining 👍🏼